tests/test_mantatail.py

```diff
@@ -151,6 +151,39 @@ def test_youre_not_on_that_channel(user_alice, user_bob):
     assert received == b":mantatail 442 #foo :You're not on that channel\r\n"
 
 
+def test_send_privmsg(user_alice, user_bob):
+    user_alice.sendall(b"JOIN #foo\r\n")
+    time.sleep(0.1)
+    user_bob.sendall(b"JOIN #foo\r\n")
+
+    while receive_line(user_alice) != b":Bob!BobUsr@127.0.0.1 JOIN #foo\r\n":
+        pass
+
+    while receive_line(user_bob) != b":mantatail 366 Bob #foo :End of /NAMES list.\r\n":
+        pass
+
+    user_bob.sendall(b"PRIVMSG #foo :Foo\r\n")
+    assert receive_line(user_alice) == b":Bob!BobUsr@127.0.0.1 PRIVMSG #foo :Foo\r\n"
+
+    user_alice.sendall(b"PRIVMSG #foo :Bar\r\n")
+    assert receive_line(user_bob) == b":Alice!AliceUsr@127.0.0.1 PRIVMSG #foo :Bar\r\n"
+
+    user_bob.sendall(b"PRIVMSG #foo :Baz\r\n")
+    assert receive_line(user_alice) == b":Bob!BobUsr@127.0.0.1 PRIVMSG #foo :Baz\r\n"
```

Comment:
Sending multiple words should be tested too, because accidentally writing `.split(" ", 2)` would be enough to break it:

```suggestion
    user_bob.sendall(b"PRIVMSG #foo :Hello world\r\n")
    assert receive_line(user_alice) == b":Bob!BobUsr@127.0.0.1 PRIVMSG #foo :Hello world\r\n"
```